packages/core/core/src/Strapi.ts

```diff
@@ -574,9 +575,17 @@ class Strapi extends Container implements Core.Strapi {
 
     this.isLoaded = true;
 
+    for (const cb of this.loadedCallbacks) {
+      await cb();
+    }
+
     return this as this & Required<Core.Strapi>;
   }
 
+  async loaded(cb: Utils.Function.Any) {
+    this.loadedCallbacks.push(cb);
+  }
```

Comment:
I don't quite get the naming, these are callbacks that will be called once loading strapi? In which context this will be used?

Reply:
yes the goal is for facades to do 

```
let instance; 

strapi.loaded(() => {
 instance = strapi.xxxx
})

export { instance }
```

Reply:
Do you think `onLoad` / `afterLoad` would be a better name? . I was a little bit confused by the name `loaded` as I would expect that to be a scalar attribute rather than a method, wdyt?